### PR TITLE
Add pricing information to public pages

### PR DESCRIPTION
**Pull request by rebeccacremona**
See ENG-370

image
image
image

### CODE REVIEW

File: perma_web/perma/templates/about.html [hunk 1]
```diff
@@ -69,9 +70,19 @@ <h2 class="body-ah">Accounts</h2>
       <p class="body-text">Organizations (such as law firms, publishers, non-profits and others) or individuals not associated with an academic institution or court are both able to use Perma via paid subscription:</p>
       <ul class="body-text" style="list-style: circle;">
         <li>Organizations can administer unlimited Perma accounts for their users for a monthly flat group rate. These registrar accounts also include collaboration tools and administrative controls.</li>
-        <li>Individuals can access Perma via tiered subscriptions that fit their particular needs.</li>
+        <li>Individuals not affiliated with a Registrar can access Perma via tiered subscriptions that fit their particular needs, or make an ad hoc purchase of links.</li>
       </ul>
-      <p class="body-text">Subscription status does not affect the preservation, access, or visibility of already-made links, just the amount of Perma Links a user can create in a given month.</p>
+      <p class="body-text"><em>Subscription status does not affect the preservation, access, or visibility of already-made links, just the amount of Perma Links a user can create in a given month</em>.</p>
```
Comment (bensteinberg) [inline perma_web/perma/templates/about.html]:
```suggestion
      <p class="body-text"><em>Subscription status does not affect the preservation, access, or visibility of already-made links, just the number of Perma Links a user can create in a given month</em>.</p>
```

File: perma_web/perma/templates/about.html [hunk 2]
```diff
@@ -69,9 +70,19 @@ <h2 class="body-ah">Accounts</h2>
...
+    </div>
+  </div>
+</div>
+
+<div class="container cont-reading cont-fixed" id="pricing">
+  <div class="row">
+    <div class="col-sm-8 docs reading-body col-right">
+      <h2 class="body-ah">Pricing</h2>
+      <p class="body-text">Organizations who do not qualify for free usage can <a href="{% url 'contact' %}">contact our team</a> to learn about creating a subscription for their organization to administer Perma.cc to their users. Pricing is based on the amount of users in an organization and the expected volume of link creation.</p>
```
Comment (bensteinberg) [inline perma_web/perma/templates/about.html]:
```suggestion
      <p class="body-text">Organizations that do not qualify for free usage can <a href="{% url 'contact' %}">contact our team</a> to learn about creating a subscription for providing Perma.cc to their users. Pricing is based on the number of users in an organization and the expected volume of link creation.</p>
```

File: perma_web/perma/templates/about.html [hunk 3]
```diff
@@ -69,9 +70,19 @@ <h2 class="body-ah">Accounts</h2>
...
+      <p class="body-text">For individuals who are not affiliated with a Registrar, there are two options for purchasing additional Perma Links beyond their trial: an ongoing subscription that releases a set amount of links to be created each month, and ad hoc purchasing of a batch of links to be created. Pricing details can be found on your Perma.cc dashboard or in our <a href="{% url 'docs' %}#usage-plans">user guide</a>.</p>
```
Comment (bensteinberg) [inline perma_web/perma/templates/about.html]:
```suggestion
      <p class="body-text">For individuals who are not affiliated with a Registrar, there are two options for purchasing additional Perma Links beyond their trial: an ongoing subscription that releases a set number of links to be created each month, and ad hoc purchasing of a batch of links. Pricing details can be found on your Perma.cc dashboard or in our <a href="{% url 'docs' %}#usage-plans">user guide</a>.</p>
```

File: perma_web/perma/templates/docs/accounts.html [hunk 4]
```diff
@@ -71,7 +71,20 @@ <h3 id="paid-usage-groups" class="body-bh">Paid Usage Plans for Groups</h3>
       <p class="body-text">Law firms, publishers, news organizations and other entities may become registrars through a paid usage plan, which will give them access to Perma's collaboration and administrative tools and allow them to make Perma available to their employees. To learn more about paid plans, please <a href="{% url 'contact' %}?upgrade=organization">contact us</a>.</p>
 
       <h3 id="paid-usage-individuals" class="body-bh">Paid Usage Plans for Individuals</h3>
-      <p class="body-text">For individuals who are not affiliated with Perma regsitrars, or who want independent access to Perma, we offer paid usage plans. We currently offer monthly subscriptions and bundled purchase options. To learn more, login and visit the <a href="{% url 'user_management_settings_usage_plan' %}">Usage Plan</a> tab in your Account Settings.</p>
+      <p class="body-text">For individuals who are not affiliated with Perma registrars, or who want independent access to Perma, we offer paid usage plans. There are two options, based on different types of usage:</p>
+      <p class="body-text">Ongoing subscription for users who create Perma Links on a consistent cadence</p>
```
Comment (bensteinberg) [inline perma_web/perma/templates/docs/accounts.html]:
```suggestion
      <p class="body-text">Ongoing subscription for users who create Perma Links at a consistent cadence</p>
```

**Review by bensteinberg — APPROVED** [review]
LGTM; consider these copyediting suggestions.

**Review by clare-stanton — APPROVED** [review]

**Pull request merged**